src/Modules/ASayModule/Templates/Announcement.mt

```diff
@@ -0,0 +1,49 @@
+﻿<component>
+    <property type="string" name="text"/>
+    <template>
+        <frame id="window" pos="-80 -47" halign="center" valign="center">
+            <frame id="textFrame" size="160 10" valign="center2" halign="left" pos="0 0" z-index="0">
+                <label id="text" size="160 10" valign="center2" halign="left" textsize="3" text="{{text}}"
+                       textfont="GameFontExtraBold" pos="0 0" textcolor="fff" z-index="1" textprefix="$t$s"/>
+            </frame>
+            <quad id="bgQuad" size="170 10" style="UICommon64_1" substyle="BgFrame1" halign="left" colorize="f05"
+                  opacity="0.7" pos="0 0" valign="center" z-index="-1"/>
+            <label size="10 10" valign="center2" halign="center" textsize="4" text="" textcolor="f05" z-index="3"
+                   pos="5 0"/>
+            <quad size="10 10" style="UICommon64_1" substyle="BgFrame1" halign="left" colorize="fff" opacity="1"
+                  pos="0 0" valign="center" z-index="2"/>
+        </frame>
+        <script>
```

Comment:
Recommended to define `<script main="true">` for scripts that contains the main function.

Reply:
Done